packages/eslint-plugin-orbit-components/babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: ["@babel/preset-env", "@babel/preset-typescript"],
+};
```

Comment:
If the only reason why we're using Babel is to have ES imports, we should convert imports to requires and get rid of the build step. Thoughts?